.github/workflows/deploy-dev.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     with:
       docker_image_version_suffix_label: dev
     secrets:
-      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+      aws_account_id: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
       aws_role_id: ${{ secrets.AWS_ROLE_ID }}
-      aws_region: ${{ secrets.AWS_REGION }}
+      aws_region: us-west-2
```

Comment:
would this be better as a secret/variable?

Reply:
Yes! But we are using a shared template. Need to keep this like that for now for compatibility purposes. Need to change all repos that use this template before doing that.